### PR TITLE
ffmpegx and tvh

### PR DESCRIPTION
**Pull request by CvH**
- update Tvh and all depends (ffmpegx)
- fixes bug at Tvh settings addon where not all ips were generated at the AVM tab
- update ffmpeg-tools as ffmpegx was updated

### CODE REVIEW

**Comment by CvH** [comment]
- updated: added comskip at multible request
testbuild if someone likes to give it a shot (especially comskip) http://cvh.libreelec.tv/addons/9.0/tvh/generic/

**Comment by CvH** [comment]
works for me :)

**Comment by CvH** [comment]
added part from ffmpeg cleanup from https://github.com/LibreELEC/LibreELEC.tv/pull/2931

File: packages/addons/service/tvheadend42/package.mk [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
+PKG_VERSION_NUMBER="4.2.6-42"
+PKG_REV="116"
 PKG_ARCH="any"
 PKG_LICENSE="GPL"
 PKG_SITE="http://www.tvheadend.org"
 PKG_URL="https://github.com/tvheadend/tvheadend/archive/$PKG_VERSION.tar.gz"
-PKG_SOURCE_DIR="tvheadend-${PKG_VERSION}*"
-PKG_DEPENDS_TARGET="toolchain avahi curl dvb-apps ffmpegx libdvbcsa libhdhomerun libiconv openssl pngquant:host Python2:host tvh-dtv-scan-tables"
+PKG_DEPENDS_TARGET="toolchain avahi comskip curl dvb-apps ffmpegx libdvbcsa libhdhomerun \
+                    libiconv openssl pngquant:host Python2:host tvh-dtv-scan-tables"
```
Comment (MilhouseVH) [inline packages/addons/service/tvheadend42/package.mk]: Not sure continuing a variable assignment over multiple lines is a good idea, as this makes grepping for stuff that bit harder. A single line for these PKG_* variables is generally the "norm" and should be maintained IMHO.

Comment (MilhouseVH) [reply packages/addons/service/tvheadend42/package.mk]: As discussed on Slack, leave it as it is and I'll merge this - we can decide what to about splitting in another PR (if necessary).

**Pull request merged**